students/surowiec_marek/lesson_03_funcions/calculator.py

```diff
@@ -0,0 +1,86 @@
+# Calculator
+def adding(add_var_1, add_var_2):
+    return(add_var_1 + add_var_2)
+
+
+def subtract(add_var_1, add_var_2):
+    return (add_var_1 - add_var_2)
+
+
+def multiply(add_var_1, add_var_2):
+    return (add_var_1 * add_var_2)
+
+
+def divide(add_var_1, add_var_2):
+    return (add_var_1 / add_var_2)
+
+
+def power(add_var_1, add_var_2):
+    return (add_var_1 ** add_var_2)
+
+
+def help():
+    print("HELP")
+    print("a - add")
+    print("s - subtract")
+    print("m - multiply")
+    print("d - divide")
+    print("p - power")
+    print("h,? - help")
+    print("q - QUIT")
+
+
+def get_operands():
+    global add_var_1
+    global add_var_2
+    print("Input 1st operand:")
+    add_var_1 = int(input())
+    print("Input 2nd operand:")
+    add_var_2 = int(input())
```

Comment:
`add_var_1` i `add_var_2` sugerują, że są to operandy dodawania, podczas gdy są wykorzystywane w różnych operacjach. Zalecam zmianę nazwy tutaj i w pozostałych przypadkach.